ox_bqpipeline/bqpipeline.py

```diff
@@ -422,10 +422,32 @@ def set_query_params(self, query_params):
             return [set_parameter(key, query_params[key]) for key in
                     query_params]
 
+    def get_query_details(self, query_details):
+        """Returns the query path, destination, parameters and GCS flag.
+
+        :param query_details <tuple|str>:
+        :returns: Query path, destination GCS path or table spec, query params,
+                and boolean to signify if the destination is GCS.
+        """
+        sql_path, destination, query_params = None, None, None
+        is_gcs_dest = False
+        if type(query_details) == tuple:
+            sql_path = query_details[0]
+            destination = query_details[1]
+            if len(query_details) > 1:
```

Comment:
see above note to decrease level of nested ifs and avoid checking len(...) == 3 when len(...) <= 1.

Reply:
Done.